src/pages/index.jsx

```diff
@@ -57,72 +58,73 @@ import ViewVolunteers from "../components/Volunteer/ViewVolunteers";
 import Layout from "../layouts/app";
 import EditRound from "../components/Rounds/Edit";
 
-if(typeof(document)!='undefined')
-  document.title="MUCAPP";
+if (typeof (document) != 'undefined')
```

Comment:
strict not equality missing

Reply:
Not my code 🤷‍♂

Reply:
Don't worry about that now guys, after this PR is merged I'm setting this repo so this kinda things don't go through without failing the build. I did a mistake a few days back, I need to fix it.